iBudget/ibudget/views.py

```diff
@@ -0,0 +1,43 @@
+"""
+Views module for handling AWS S3
+"""
+from django.http import HttpResponse
+from django.views import View
+from utils.aws_helper import upload, get_image_url, del_photo, check_if_in_bucket
+from utils.response_helper import (RESPONSE_400_INVALID_DATA)
+
+
+class AwsCall(View):
+    """ View for handling CRUD methods for images in the AmazonS3 bucket
+    """
+
+    def post(self, request):
+        """The name property of the file which is passed is 'icon', so in HTML form it must be set:
+        <input type='file' name = 'icon'>"""
+        file = request.FILES['icon']
```

Comment:
What if there is no `file` in `FILES`?

Reply:
`file` is a so-called reserved word so it can't be used for the variable name.

Reply:
Thanks.
I will change the name of the variable and modify the view to include cases when no files were provided.